### PR TITLE
README.md: replace examples with kelsey's tutorial

### PR DESCRIPTION
**Pull request by runcom**
Get rid of our examples in favor of @kelseyhightower's tutorial

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by cyphar** [comment]
<3 to @kelseyhightower, that tutorial was really great!

**Comment by cyphar** [comment]
LGTM

**Comment by feiskyer** [comment]
Though the tutorial is excellent, it's a little long for starters. I think we still need a short one in RADME and make a link to the detailed tutorial.

**Comment by runcom** [comment]
> Though the tutorial is excellent, it's a little long for starters. I think we still need a short one in RADME and make a link to the detailed tutorial.

right, though, my feel was that we can't just have a README full of examples. Instead, it would be great to provide examples in man pages. So, the README stays tiny but people installing this would have a great time reading the man pages

**Comment by sameo** [comment]
LGTM

**Comment by mrunalp** [comment]
How about we move Kelsey's tutorial to a markdown file in this repo (with his permission)? That way it could stay up to date.

**Comment by runcom** [comment]
@mrunalp would be great

**Comment by cyphar** [comment]
Yup. Just make sure it's he's fine with it being licensed under Apache 2.0.

**Comment by mikebrow** [comment]
whoah docs!  LGTM

**Comment by kelseyhightower** [comment]
LGTM

Let me know if I need to make any adjustments on my side.

**Comment by runcom** [comment]
just moved Kelsey's tutorial to `tutorial.md` in this repo (didn't want to clutter `docs`) - let me know of any adjustments you guys need.

**Comment by jawnsy** [comment]
+1 to a separate tutorial file. I think there's two separate use cases here:

1. I'm a newbie and I want to learn cri-o -> tutorial
1. I forgot a command to do something -> a CLI reference (the readme was useful for this before, but those contents have been removed now)

I don't know whether such a reference is useful enough to maintain separate docs, though, when the tutorial is pretty good on its own. Thoughts?

File: tutorial.md [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+First we need to setup a Pod sandbox using a Pod configuration, which can be found in the `cri-o` source tree: 
+
+```
+cd $GOPATH/src/github.com/kubernetes-incubator/cri-o
+```
+
+Next create the Pod and capture the Pod ID for later use:
+
+```
+POD_ID=$(sudo ocic pod create --config test/testdata/sandbox_config.json)
```
Comment (mikebrow) [inline tutorial.md]: it's "pod start" now vs "pod create"

**Comment by runcom** [comment]
> I forgot a command to do something -> a CLI reference (the readme was useful for this before, but those contents have been removed now)

@jawnsy this use case belongs to the `man` - not to the README here I guess

**Comment by runcom** [comment]
I've rebased this and fixed @mikebrow comment, @mrunalp PTAL

File: tutorial.md [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+{
+    "cniVersion": "0.2.0",
+    "type": "loopback"
+}
+EOF'
+```
+
+At this point `cni` is installed and configured to allocation IP address to containers from the `10.88.0.0/16` subnet.
+
+### docker
```
Comment (mrunalp) [inline tutorial.md]: We should take out this section as it isn't required anymore.

Comment (runcom) [inline tutorial.md]: fixed

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**